src/components/v2/Layout/Header/index.tsx

```diff
@@ -21,8 +21,8 @@ export const Header = ({ pageTitle }: IHeaderProps) => {
           {pageTitle}
         </Typography>
         <Box flexDirection="row" display="flex" flex={1} justifyContent="right">
-          <XvsCoinInfo css={styles.rightItemPaper} />
-          <VaiCoinInfo css={styles.rightItemPaper} />
+          <XvsCoinInfo css={styles.rightItemPaper} className="coinInfo" />
+          <VaiCoinInfo css={styles.rightItemPaper} className="coinInfo" />
```

Comment:
I can't find the `coinInfo` class being defined anywhere, can it be removed or am I missing something?

Reply:
yeah this was the issue we talked about [here](https://github.com/VenusProtocol/venus-protocol-interface/pull/214#pullrequestreview-917593692) with emotion needing class to grab to process the css prop :/ I agree it is a bit confusing

Reply:
In this case I moved the styles to the parent in order to use the first and last  child selectors but we will still likely see classNames appear like this unless we choose a different solution

Reply:
But what I mean is that the `coinInfo` class isn't defined anywhere in CSS, so it seems it's doing nothing here.
I'm not sure I understand the issue to be honest, since if you want to pass specific styling to these components you could pass it through the `css` property.

Reply:
That references a property of an object, it wouldn't translate to the class name `coinInfo` (it would be formatted into a unique name).

Reply:
It is required for emotion to process the css prop. I linked the thread where we discussed this previously

Reply:
Ok, I thought this discussion was about just passing `className` (as in the prop, not an actual string) but fair enough!